libs/react-components/src/lib/button/button.tsx

```diff
@@ -93,7 +88,7 @@ export function GoAButton({
       type={type}
       size={size}
       variant={variant}
-      disabled={disabled}
+      {...(disabled ? { disabled: true } : {})}
```

Comment:
```suggestion
      disabled={typeof value === 'undefined' ? undefined : value ? "true" : "false"}
```
Suggesting this change to allow for a) true, b) false, or c) undefined. if prop not passed in it will not include it in the DOM